metadata-ingestion/tests/unit/test_great_expectations_action.py

```diff
@@ -248,6 +337,46 @@ def test_DataHubValidationAction_basic(
     )
 
 
+@mock.patch("datahub.emitter.rest_emitter.DatahubRestEmitter.emit_mcp", autospec=True)
+def test_DataHubValidationAction_pandas(
+    mock_emitter: mock.MagicMock,
+    ge_data_context: DataContext,
+    ge_validator_pandas: Validator,
+    ge_validation_result_suite_pandas: ExpectationSuiteValidationResult,
+    ge_validation_result_suite_id_pandas: ValidationResultIdentifier,
+) -> None:
+    server_url = "http://localhost:9999"
+
+    datahub_action = DataHubValidationAction(
+        data_context=ge_data_context,
+        server_url=server_url,
+        platform_instance_map={"my_df_datasource": "custom_platefrom"},
+    )
+
+    assert datahub_action.run(
+        validation_result_suite_identifier=ge_validation_result_suite_id_pandas,
+        validation_result_suite=ge_validation_result_suite_pandas,
+        data_asset=ge_validator_pandas,
+    ) == {"datahub_notification_result": "DataHub notification succeeded"}
+
+    mock_emitter.assert_has_calls(
+        [
+            mock.call(
+                mock.ANY,
+                MetadataChangeProposalWrapper(
+                    entityType="assertion",
+                    changeType="UPSERT",
+                    entityUrn="urn:li:assertion:7e04bcc3b85897d6d3fef6c998db6b05",
+                    aspectName="dataPlatformInstance",
```

Comment:
there should be calls for other aspects here as well - e.g. assertionInfoClass

Reply:
Ok i will fix it 👍

Reply:
fixed